### ISSUE TITLE
Leak 

### ISSUE THREAD

**Issue by matsumoto79** [brow/leaves#15]
Hi Brow, thanks for sharing your projects .

I tested your project in a real device (Iphone4) with 12 more images in the Image Example mode and I think uncache you are doing is not sufficient, the images are kept in memory in another step. If you run Instruments/Allocations you can see how the liveBytes grow until you have gone throw all the images. You can see some info about this object's allocation, like Responsible Library = ImageIO or Responsible Caller = ImageIO_Malloc. However I cant fix it so far ... your help would be really appreciate it.

Thanks

**Comment by jeffh2170** [comment]
Bump

I am seeing the same problem too.  When zooming in and out several times and flipping pages causes a huge memory leak or huge allocations.  I've tried running Instruments and cannot pinpoint where the leak is taking place... Anyone else have any thoughts or ideas?  BTW I'm using Hammerlyrodrigo's most recent version with zooming enabled.

**Comment by jeffh2170** [comment]
matsumoto79, I had received a solution (test it and verify for yourself!) to fix the memory problem I was having from Rodrigo Hammerly, assuming you are using his version (September 2011).  For me it worked for the most part, I just had to tweak some of the tile settings.  I changed the level of detail down to 4 and the bias to 2, and it seemed to work fine.  The only time it didn't work was I had a higher detail and bias, which I think just took up too much memory to cache.

Add these two lines of code to -(void)getImages in the LeavesView.m file.
topPageZoomLayer.contents = nil;
topLeftPageZoomLayer.contents = nil;
